packages/vuetify/src/effects/__tests__/color.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { reactive, computed } from 'vue'
+
+import {
+  ColorProps,
+  useTextColor,
+  useBackgroundColor,
+} from '../color'
+
+describe('color.ts', () => {
```

Comment:
I really like this approach. Does anyone @vuetifyjs/core-contributors @vuetifyjs/core-team object to this being documented as a standard for writing tests for effects (and any similar functionality).

To clarify, I'm referring to the nested describes.

Reply:
I think nested describes are very good thing and they help keeping tests organized.

Reply:
Yeah I also use nested describes a lot, in my views for example where I have form validations.

    describe MyView.vue
        describe form validations
            test field is required

Reply:
Might be ok for smaller files, but for bigger maybe just split the file into smaller ones? We're not restricted to having 1 file per `__test__`

Reply:
Agreed with jacek, it's a case-by-case decision I think.

Reply:
To clarify, this was more for effects and similar files such as helper and utility ones.